### ISSUE TITLE
[🐛 Bug]: Chrome V111 issue happening in dotnet project aswell. 

### ISSUE THREAD

**Issue by uday0804** [SeleniumHQ/selenium#11790]
### What happened?

I have tried remote all origins = * in chrome options in my dotnet project. But It doesn't worked out. All my tests are failing with OpenQa.Selenium webdriver application cache threw a new exception of type system invalid operation exception.

Help is really appreciated.

### How can we reproduce the issue?

```shell
Update Chrome to latest version 111 and run tests using selenium and C#.
```

### Relevant log output

```shell
NA
```

### Operating System

Windows 10

### Selenium version

.Net 4.6 or latest

### What are the browser(s) and version(s) where you see this issue?

Chrome V111

### What are the browser driver(s) and version(s) where you see this issue?

chrome driver 111

**Comment by diemol** [comment]
Closing as we did not get more information.

**Comment by ajahongir** [comment]
@diemol guys. need your help. dont want to put new issue. just fast question: after updating browser upto v112 now devtools latest version is v111, so how long need to wait to release v112 devtools? or any other solutions?

**Comment by titusfortner** [comment]
Chrome v111 devtools should still work for v112 (you just get the warning). Lots of stuff has been added to trunk and we have to make sure everything is good before we can do the next release. (and by we I mean @diemol)

**Comment by ajahongir** [comment]
actually devtools running but having some issues(can not figureout) BUT as I can see. devtools does made for every single version of chrome. and i suggest it will not will be available for v112 and i decide to downgrade my chrome browser and solve an issue. anyway, thank you guys for fast response and support!
ps: before long time i have beed using firefox, and didnt know about chromdirever and devtools - i can say - its perfect. its what i can not dreams, we could name it **dreamtool** :)
